packages/zipkin/src/tracer/index.js

```diff
@@ -39,6 +39,8 @@ class Tracer {
     localEndpoint,
     /* eslint-disable no-console */
     log = console,
+    // defaultTags allows the user to pass tahs that should be added to all spans by
```

Comment:
I'd tighten it up though curious how this question came up.. where did they think the tags would go? maybe it can help revise the comment.

"defaultTags custom tags to add to all sampled spans. defaults to empty"

Reply:
I like documenting my issues so I was grabbing references to open https://github.com/hypertrace/agent-config/issues/21 then I realised some params have comments and some don't and hence I felt the need to improve this.